lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -14,6 +14,7 @@
 import dogstats_wrapper as dog_stats_api
 import re
 import unicodecsv
+from capa.responsetypes import StudentInputError, ResponseError, LoncapaProblemError
```

Comment:
This felt wrong. We should probably just swallow the exception at the CAPA layer, yeah?